src/Parser/Shaman/Restoration/Modules/Spells/HealingSurge.js

```diff
@@ -15,6 +15,26 @@ class HealingSurge extends Analyzer {
     combatants: Combatants,
   };
 
+  get suggestedThreshold(){
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.